assembly/assembly-wsmaster-war/pom.xml

```diff
@@ -235,6 +235,10 @@
             <groupId>org.eclipse.che.infrastructure</groupId>
             <artifactId>infrastructure-openshift</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.che.multiuser</groupId>
+            <artifactId>che-multiuser-api-authentication-commons</artifactId>
```

Comment:
Why do we need this change? If you remove che-multiuser-api-authentication-commons nothing will change. War is compiled, che-multiuser-api-authentication-commons.jar will appear in the lib.